controllers/operands/networkAddons_test.go

```diff
@@ -632,6 +639,37 @@ var _ = Describe("CNA Operand", func() {
 			}
 		}
 
+		kubevirtIpamTester := func(o ipamAnnotationParams) {
+			existingCNAO, err := NewNetworkAddons(hco)
+			Expect(err).ToNot(HaveOccurred())
+			if o.ipamExists {
+				existingCNAO.Spec.KubevirtIpamController = &networkaddonsshared.KubevirtIpamController{}
+			}
+
+			if o.setFeatureGate {
+				hco.Spec.FeatureGates.DeployKubevirtIpamController = ptr.To(o.featureGateValue)
+			}
```

Comment:
I think we can do it with one boolean: just pass the required FG value to the test, and set it unconditionally.

Reply:
but then it means we set even if false, overriding the default value
it can work of course, i just copy pasted the from the other tests

Reply:
I won't hold the PR for that. your call

Reply:
we even test it "should not have KIC if feature gate does not exist"
i prefer to keep as is